src/components/DeckMechanisms/index.js

```diff
@@ -395,7 +392,7 @@ export default class DeckMechanisms extends React.Component {
     const card = this.state.deck.find(card => card.id === id)
     const isAffordable = card.mana <= this.state.mana
     const canBePlayed = !(
-      this.state.turn === 1 && ['W1', 'I3', 'F4', 'N15'].includes(id)
+      this.state.turn === 1 && ['W1', 'S10', 'F4', 'N15'].includes(id)
```

Comment:
Broken Truce can't be played on turn 1

Reply:
If the second played has 4 mana on turn 1, this should include an additional check - `const canBePlayed = !(this.state.playerOrder === 'FIRST' && this.state.turn ===1 && ['W1', 'S10', 'F4', 'N15'].includes(id))`